options/locale/locale_en-US.ini

```diff
@@ -868,6 +868,8 @@ already_forked = You've already forked %s
 fork_to_different_account = Fork to a different account
 fork_visibility_helper = The visibility of a forked repository cannot be changed.
 use_template = Use this template
+clone_https_description = Use Git or checkout with SVN using the web URL.
+clone_ssh_description = Use a password protected SSH key.
```

Comment:
It looks like these were copied from GitHub, which is fine, but afaik we don't support SVN like GH does.

Reply:
Yes they were, I didn't come up with a better one

Reply:
I've added a commit that aims to fix that description